charmcraft/commands/build.py

```diff
@@ -225,6 +242,22 @@ def run(self, bases_indices: Optional[List[int]] = None) -> List[str]:
 
         return charms
 
+    def build_charm_in_instance(self, instance: Executor, bases_index: int) -> str:
```

Comment:
I would use a name different than build to not confuse in the future when parts lifecycle is introduced

Reply:
How about s/build/pack?

Reply:
Done, renamed to pack_charm_in_instance()